app/assets/stylesheets/admin.scss

```diff
@@ -177,3 +177,9 @@
 .bulk-operation-action-button-group {
   margin-bottom: 0;
 }
+
+.admin-application-card {
+  h2.govuk-summary-card__title {
+    font-size: 1.5rem;
+  }
+}
```

Comment:
Whats the motivation for not following the design system styling?

Reply:
Could this do something like `@include`, `@extend` - can't remember which - one of the override classes from the design system?

https://design-system.service.gov.uk/styles/font-override-classes/